editor/editor_export.cpp

```diff
@@ -274,8 +274,6 @@ void EditorExportPlatform::gen_debug_flags(Vector<String> &r_flags, int p_flags)
 }
 
 Error EditorExportPlatform::_save_pack_file(void *p_userdata, const String &p_path, const Vector<uint8_t> &p_data, int p_file, int p_total) {
-	if (p_path.ends_with(".so") || p_path.ends_with(".dylib") || p_path.ends_with(".dll"))
-		return OK;
```

Comment:
cc @karroffel @endragor Follow up from https://github.com/godotengine/godot/pull/11580#pullrequestreview-97189941. I don't know if this will break anything.